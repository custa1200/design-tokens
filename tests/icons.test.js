const fs = require('fs-extra');
const path = require('path');

describe('icons ', () => {
  test('snapshot should be matched', () => {
    const iconsArr = fs.readdirSync(
      path.join(__dirname, '../src/assets/icons/')
    );
    expect(iconsArr).toMatchSnapshot();
    expect(iconsArr.length).toMatchSnapshot();
  });
});
