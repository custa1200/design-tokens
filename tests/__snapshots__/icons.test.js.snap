// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons  snapshot should be matched 1`] = `
Array [
  "delete.svg",
  "edit.svg",
  "restart.svg",
  "search.svg",
  "settings.svg",
  "trash.svg",
]
`;

exports[`icons  snapshot should be matched 2`] = `6`;
